README.md

```diff
@@ -36,32 +36,32 @@ In this city ladies look pretty
 Guys tell jokes so they can seem witty
 Tell a funny joke just to get some play
 Then you try to make a move and she says, "No way"
-Girls are fakin' goodness sakin'
+Girls are faking goodness saking
 They want the man who brings home the bacon
 Got no money and you got no car
 Then you got no women and there you are
 Some girls are sadistic, materialistic
-Lookin' for a man makes them opportunistic
-They're lyin' on a beach perpetrating a tan
+Looking for a man makes them opportunistic
+They're lying on a beach perpetrating a tan
 So a brother with the money can be their man
-So on the beach you're strollin' real high rollin'
+So on the beach you're strolling real high rolling
 Everything you have is yours and not stolen
-A girl runs up with somethin' to prove
+A girl runs up with something to prove
 So don't just stand there bust a move
 
 Your best friend Harry has a brother Larry
 In five days from now he's gonna marry
-He's hopin' you can make it there if you can
+He's hoping you can make it there if you can
 Cause in the ceremony you'll be the best man
 You say neat-o, check your libido
 And roll to the church in your new tuxedo
 The bride walks down just to start the wedding
 And there's one more girl you won't be getting
-So you start thinkin' then you start blinking
+So you start thinking then you start blinking
 The bridesmaid looks and thinks that you're winking
 She thinks your kinda cute so she winks back
-And now your feelin' really fine cause the girl is stacked
-Reception's jumpin' bass is pumpin'
-You look at the girl and your heart starts thumpin'
+And now your feeling really fine cause the girl is stacked
+Receptiongs jumping bass is pumping
```

Comment:
** Receptiongs** is not a word. Hahaha. I think search and replace went a little wild here.